res/values/themes.xml

```diff
@@ -137,4 +137,10 @@
     <item name="emoji_button_bg">#ffffff</item>
     <item name="emoji_color">#000000</item>
   </style>
+  
+<style name="Darkv2" parent="@style/BaseTheme"><item name="android:isLightTheme">false</item><item name="colorKeyboard">#ff121212</item><item name="colorKey">#ff434343</item><item name="colorKeyActivated">#ff1a237e</item><item name="colorLabel">#ffffffff</item><item name="colorLabelActivated">#ff000000</item><item name="colorLabelLocked">#ffffffff</item><item name="secondaryLightOffset">-0.2</item><item name="colorSubLabel">#ffaaaaaa</item><item name="keyBorderWidth">0.0dip</item><item name="keyBorderWidthActivated">0.0dip</item><item name="keyBorderColorBottom">#ff121212</item><item name="emoji_button_bg">?colorKeyActivated</item><item name="emoji_color">#ffffffff</item></style>
+<style name="Lightv2" parent="@style/BaseTheme"><item name="android:isLightTheme">true</item><item name="colorKeyboard">#fff2f5ff</item><item name="colorKey">#ffffffff</item><item name="colorKeyActivated">#ffc2e7ff</item><item name="colorLabel">#ff222222</item><item name="colorLabelActivated">#ff000000</item><item name="colorLabelLocked">#ffffffff</item><item name="secondaryLightOffset">0.0</item><item name="colorSubLabel">#ff333333</item><item name="keyBorderWidth">0.0dip</item><item name="keyBorderWidthActivated">0.0dip</item><item name="keyBorderColorLeft">#ffcccccc</item><item name="keyBorderColorTop">#ffeeeeee</item><item name="keyBorderColorRight">#ffcccccc</item><item name="keyBorderColorBottom">#ffaaaaaa</item><item name="emoji_button_bg">?colorKeyActivated</item><item name="emoji_color">#ff000000</item></style>
+<style name="Amoled" parent="@style/Darkv2"><item name="android:isLightTheme">false</item><item name="colorKeyboard">#ff000000</item><item name="colorKey">?colorKeyboard</item><item name="colorKeyActivated">#ff222222</item></style>
```

Comment:
`Black` and `AltBlack` are already intended for Amoled screens. Could you suggest improvements rather than create a third black theme ?

Reply:
Well, it's just an improvement of the theme, I added amoled just to identify it 😅, these themes are just redesigns, those you don't like can be discarded